lib/model/katex.dart

```diff
@@ -136,6 +154,110 @@ class _KatexParser {
   KatexNode _parseSpan(dom.Element element) {
     // TODO maybe check if the sequence of ancestors matter for spans.
 
+    final spanClasses = List<String>.unmodifiable(element.className.split(' '));
+
+    if (element case dom.Element(localName: 'span', :final className)
+        when className.startsWith('vlist')) {
+      switch (element) {
+        case dom.Element(
+          localName: 'span',
+          className: 'vlist-t',
+          attributes: final attributesVlistT,
```

Comment:
I wonder what "t" from "vlist-t" stands for. The "r" from "vlist-r" I assume is for "row" from the naming the `KatexVlistRowNode`.

Reply:
I think the "t" is probably for "table". In the CSS, the `.vlist-t` is the element that gets `display: inline-table`:
https://github.com/KaTeX/KaTeX/blob/2fe1941b/src/styles/katex.scss#L183-L184
so that its internal layout is that of a table.